tests/index.html

```diff
@@ -16,6 +16,13 @@
 
     {{content-for "head-footer"}}
     {{content-for "test-head-footer"}}
+
+    <script>
+      // testdouble support
+      // testdouble doesn't natively support the browser as it assumes
+      // that a `global` identifier will be available, but that is not the case for browsers
```

Comment:
I reported this here: https://github.com/testdouble/testdouble.js/issues/438
and it seemed there was 0 interest in resolving... :-\

Reply:
I also opened a new issue: https://github.com/testdouble/testdouble.js/issues/471

to see what they feel like doing about it 🤷